### PR TITLE
december

### PR DESCRIPTION
**Pull request by craig-mulligan**
@imrehg what do you think of the 'Forum thread of the month' callout? 

Also anything we should add/remove?

### CODE REVIEW

**Comment by imrehg** [comment]
Good section! Maybe add this thread as well, it was a good tech conversation https://forums.resin.io/t/raspberry-pi-3-and-additional-network-dongle/438

File: src/data/featured.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-	"title": "Samsung ARTIK 710 Comes to resin&#173;.io",
-	"link": "https://resin.io/blog/samsung-artik-710/",
-	"image": "https://resin.io/blog/content/images/2016/10/ARTIK_710_blog.jpg",
-  "text": "The ARTIK 710 is also the first board on resin&#173;.io that is available with the latest resinOS 2.0",
-	"button": "Learn more"
+	"title": "Keeping Your System Running with a Host OS Watchdog",
+	"text": "We are adding a watchdog feature, which helps keep your devices in a healthy state!",
+	"link": "https://resin.io/blog/keeping-your-system-running-watchdog/",
+	"image": "https://resin.io/blog/content/images/2016/12/Download_1_24.png",
```
Comment (craig-mulligan) [inline src/data/featured.json]: @imrehg do you think this is good for the feature article? Is there any image we could attach to it? The current one isn't v eye catching.

Comment (imrehg) [inline src/data/featured.json]: I think it's a good feature article, the image is a tough question, it's not a very visual topic. Some ideas on the watchdog thought (crop them as nice):

- [Pompeii antique watchdog](https://upload.wikimedia.org/wikipedia/commons/thumb/f/fd/Pompeii_-_Casa_di_Paquius_Proculus_-_Watchdog.jpg/898px-Pompeii_-_Casa_di_Paquius_Proculus_-_Watchdog.jpg)
- [Snow and doggie](https://c4.staticflickr.com/8/7294/16542677171_9a2fd17975_b.jpg)
- [going all in with cute](https://pixabay.com/p-316459/?no_redirect)
- [getting tough](https://pixabay.com/p-705991/?no_redirect)

(all above should be free to use, from google image search, free to use with modifications, `watchdog`)

Or something completely different?

Comment (craig-mulligan) [inline src/data/featured.json]: Shoo, tough one. Let's go with the cute

Comment (imrehg) [inline src/data/featured.json]: Sounds good! 🐶

**Comment by craig-mulligan** [comment]
Comments from @alisondavis17:
-Date needs to be December 
-I think we updated our events calendar and the next upcoming event is ELC in February 
-Do we have any open positions to share?

**Comment by apostolism** [comment]
Comment: Events/Monki Gras, we're not sponsoring the event. I don't think it should be in the newsletter.
Suggestion: should we add a resinOS and/or Etcher part in the newsletter?

File: src/data/posts.json [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
+    "text": "Docker related best practices, tips, and tricks which can dramatically improve the resin.io developer experience.",
+    "link": "https://resin.io/blog/our-dockerfile-tips-tricks/",
+    "button": "Learn more"
   },
   {
-    "title": "Staged rollouts POC on the resin&#173;.io Forum",
-    "text": "<a href='https://twitter.com/imrehg'>@imrehg</a> has put together a script that leverages update locking to issue updates to subset of your applications device.",
-    "link": "https://talk.resin.io/t/staged-roll-out-helper-script/368",
-    "button": "Learn more"
+    "title": "Interact with resin supervisor via node-red",
```
Comment (imrehg) [inline src/data/posts.json]: Maybe capitalization?
`Interact with the resin.io Supervisor via Node-RED`

File: src/data/posts.json [hunk 3]
```diff
@@ -1,31 +1,32 @@
...
+    "title": "Use the API, Luke!",
+    "tags": ["tutorial"],
+    "text": "Build the tools you need but don't yet exist with the resin.io API.",
+    "link": "https://resin.io/blog/use-the-api-luke/",
+    "image": "https://resin.io/blog/content/images/2016/12/Luke_Skywalker-1.jpg",
     "button": "Check it out"
   },
   {
-    "title": "Microsoft Azure IoT with resin&#173;.io",
+    "title": "Resin.io's system time explained",
```
Comment (imrehg) [inline src/data/posts.json]: Maybe capitalization?
`Resin.io's System Time Explained`

**Comment by imrehg** [comment]
@apostolism I think we also list events where we are going / participating in the sense of "you can meet our team here". Just saying, not to put it in the newsletter, but definitely should be on the community site.

@craig-mulligan dropped in some comments on the changes. Cheers!

**Comment by apostolism** [comment]
@imrehg let's clarify this at the events call as I had a similar discussion with @alisondavis17 a couple of days before.

**Comment by craig-mulligan** [comment]
Okay so remove monki grass? We've released new etcher versions but i don't think there is anything to link to really?

**Comment by craig-mulligan** [comment]
btw I really like this approach for feedback :)

**Comment by craig-mulligan** [comment]
Okay @alisondavis17 @imrehg @apostolism. I've scheduled the nw for 3pm on thursday (in users tz). 

Preview: http://us3.campaign-archive1.com/?u=87871f17defba2d203db77a92&id=fe6451b77d

**Comment by imrehg** [comment]
Re: etcher release last month, @jviotti did a forum announcement that is linkable: https://forums.resin.io/t/etcher-1-0-0-beta17-release/426

**Comment by craig-mulligan** [comment]
Okay updated :) Please take one last peak.

**Pull request merged**